client/src/styles/EmojiPicker.scss

```diff
@@ -0,0 +1,36 @@
+.emoji-picker {
+  width: 235px;
+  background: #fff;
+  box-sizing: border-box;
+  border-radius: 2px;
+  overflow: auto;
+  z-index: 2;
+  position: absolute;
+  bottom: 44px;
+  background-color: rgb(64, 64, 64);
+  box-shadow: 1px 2px 2px 1px rgba(0,0,0,0.3);
+
+  ul {
+      list-style-type: none;
+      margin: 0 auto;
+      padding: 5px;
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+
+      li {
+          padding: 4px;
+          margin: 0;
+          text-align: center;
+          display: flex;
+          img {
+              margin: 0 auto;
+          }
+      }
+  }
+}
+
+.selected {
+    background-color: rgb(92, 92, 92);
+    box-shadow: 1px 0px 1px 1px rgba(0,0,0,0.15);
```

Comment:
There are currently no shadows in the channel and I'd like to keep it that way for now.